_pages/how-we-work/software.md

```diff
@@ -24,6 +26,16 @@ If the name pops up, it means GSA has already purchased it. If your software is
 
 _Note: If you're looking for MS Office, see [this page]({{site.baseurl}}/office/)._
 
+## Approvals
+
+All software in use at GSA needs to be approved through the IT Standards process. There are a few exceptions:
+
+- Free and open source development packages
+  - In general, open source development libraries are exempt from IT Standards review.
+  - See the [CIO GSA Information Technology (IT) Standards Profile](<https://www.gsa.gov/directive/gsa-information-technology-(it)-standards-profile>) (section 2c) for more details.
+- [Social media](https://www.gsa.gov/about-us/newsroom/social-media/gsa-social-media-policy)
```

Comment:
I'm reluctant to put this in the handbook because one of the last discussions with Bo this was specifically mentioned; also, that policy doesn't actually mention anything about the software used for Social Media.

Reply:
Should I just take out that line for now?

Reply:
Ya, I think that would be good until we get that figured out.

Reply:
Done.